app/src/main/java/com/github/se/studybuddies/data/Group.kt

```diff
@@ -2,10 +2,17 @@ package com.github.se.studybuddies.data
 
 import android.net.Uri
 
-data class Group(val uid: String, val name: String, val picture: Uri, val members: List<String>) {
+data class Group(
+    val uid: String,
+    val name: String,
+    val picture: Uri,
+    val members: List<String>,
```

Comment:
Shouldn't we have `val members: List<User>` instead of a list of String ? I understand that we need to make an extra step to get/store in the database, but it should help to avoid making extra steps after. And for me it makes more sense.

Reply:
Nope, the firebase can only store simple types, the list of members is just their ids